src/main/java/ca/wolfram/beta/symath/MathUtils.java

```diff
@@ -15,6 +15,14 @@ public static boolean isConstantInt(MathNode node) {
         return eval == (long) eval;
     }
 
+    public static boolean isConstantInt(MathNode node, long value) {
+        return isConstantInt(node) && node.eval(null) == value;
+    }
+
+    public static boolean isReciprocal(MathNode node) {
```

Comment:
I feel like might be a function that we can move to the DivideNode, otherwise i am afraid MathUtils will be a really big file.